veracruz-utils/src/runtime_manager_message.rs

```diff
@@ -35,68 +32,77 @@ pub enum VMStatus {
 // Command-and-control messages for the runtime manager.
 ///////////////////////////////////////////////////////////////////////////////
 
-/// An enumerated type describing messages passed between to/from the Runtime
-/// Manager enclave (These originate from the Untrusted Pass-through (Veracruz
-/// server)
+/// An enumerated type describing request messages passed to the Runtime
+/// Manager enclave (these originate from the Untrusted Pass-through (Veracruz
+/// server).
 /// These messages are inteded to be serialized using bincode before transport,
-/// and deserialized using bincode after transport
+/// and deserialized using bincode after transport.
 #[derive(Serialize, Deserialize, Debug)]
-pub enum RuntimeManagerMessage {
-    /// A message generated by an operation that did not return data, but did
-    /// return a status.  Most operations return data, but if they fail, they
-    /// will return a status set to `VMStatus::Fail` (or
-    /// `VMStatus::Unimplemented` if it is not implmeneted).  Parameters in
+pub enum RuntimeManagerRequest {
```

Comment:
Should we pick another name? Cf. https://github.com/veracruz-project/veracruz/issues/374#issuecomment-1084528607